tls.yml

```diff
@@ -0,0 +1,9 @@
+tls:
+  certificates:
+    - certFile: /certs/your-crt.pem
+      keyFile: /certs/your-key.pem
+  stores:
+    default:
+      defaultCertificate:
+        certFile: /certs/your-crt.pem
+        keyFile: /certs/your-key.pem
```

Comment:
I don't think this stub file should be added, because it can't be used as it is.
I would ask you to add instructions in the documentation on how to configure the use of custom certificates, and it seems reasonable to me to describe an example traefik tls configuration file in the documentation in the same place.

Reply:
I will remove this file and add a document